src/components/utils/ZoomImage.module.css

```diff
@@ -0,0 +1,19 @@
+html:has(.zoomModal) {
```

Comment:
I thought we were using Tailwind CSS?

Reply:
We are, but it doesn't mean that we don't do CSS, we use both.

For example for the code-blocks, I want to use more CSS to avoid repetitive long class names that create a huge HTML.

In this case, Tailwind doesn't have a way to do `has()` without going through the config and I don't want a custom selector just for this small CSS.

Reply:
My advice is to only put CSS in one single file at the root because you could end up with some issues with style loading.

Also we could do most of it using Tailwind and for what is not supported do it in the CSS file at the root.